lib/iex/lib/iex/helpers.ex

```diff
@@ -1092,6 +1094,31 @@ defmodule IEx.Helpers do
     )
   end
 
+  @doc """
+  Creates a Reference from `string`.
+
+  ## Examples
+
+      iex> ref("0.21.32.43")
+      #Reference<0.21.32.43>
+
+  """
+  def ref(string) when is_binary(string) do
+    :erlang.list_to_ref('#Ref<#{string}>')
+  end
+
+  def ref(w, x, y, z)
+      when is_integer(w) and w >= 0 and is_integer(x) and x >= 0 and is_integer(y) and y >= 0 and
+             is_integer(z) and z >= 0 do
+    :erlang.list_to_ref(
```

Comment:
How does this implementation sound for `ref/4` ?
```elixir
ref("#{w}.#{x}.#{y}.#{z}")
```

Reply:
it's much more readable! though it should be a charlist, no?

```elixir
:erlang.list_to_ref('#{w}.#{x}.#{y}.#{z}')
```